packages/plugin-dev/src/node/hosted-instance-manager.ts

```diff
@@ -97,8 +99,7 @@ delete PROCESS_OPTIONS.env.ELECTRON_RUN_AS_NODE;
 @injectable()
 export abstract class AbstractHostedInstanceManager implements HostedInstanceManager {
     protected hostedInstanceProcess: cp.ChildProcess;
-    protected processOptions: cp.SpawnOptions;
```

Comment:
Why is `processOptions` removed?

Reply:
it is not used. should i keep it anyway?

Reply:
Same reason as https://github.com/theia-ide/theia/pull/5608#discussion_r298605017

Reply:
as this member is never initialized or used I guess no one should use it. Updated the changelog.

Reply:
You cannot know it. Theia is a framework, there are plenty clients ouside of this repo. Please keep changes to related changes. It seems to be one-liner to use `FileUri.fsPath` instead of `toString`.